{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Visualizing our networks"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "import os\n",
    "import pybel\n",
    "import pybel_jupyter"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'0.2.0'"
      ]
     },
     "execution_count": 2,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "pybel_jupyter.get_version()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'0.13.1'"
      ]
     },
     "execution_count": 3,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "pybel.get_version()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [],
   "source": [
    "HERE = os.path.realpath('__file__')\n",
    "NOTEBOOK_DIRECTORY = os.path.abspath(os.path.join(HERE, os.pardir))\n",
    "PROJECT_DIRECTORY = os.path.abspath(os.path.join(NOTEBOOK_DIRECTORY, os.pardir))\n",
    "\n",
    "CURATION = os.path.join(PROJECT_DIRECTORY, 'curation')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [],
   "source": [
    "IGF1R = os.path.join(CURATION, 'IGF1R', 'first_group_pickle.json')"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "collapsed": true
   },
   "source": [
    "# IGFR1"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/javascript": [
       "function init_d3_force(d3, graph, chart, width, height, function_colors) {\n",
       "    var focus_node = null;\n",
       "    var highlight_node = null;\n",
       "\n",
       "    // Highlight color variables\n",
       "\n",
       "    // Highlight color of the node boundering\n",
       "    const highlight_node_boundering = \"#4EB2D4\";\n",
       "\n",
       "    // Highlight color of the edge\n",
       "    const highlighted_link_color = \"#4EB2D4\";\n",
       "\n",
       "    // Text highlight color\n",
       "    const highlight_text = \"#4EB2D4\";\n",
       "\n",
       "    // Size when zooming scale\n",
       "    var size = d3.scalePow().exponent(1)\n",
       "        .domain([1, 100])\n",
       "        .range([8, 24]);\n",
       "\n",
       "    // Simulation parameters\n",
       "    const linkDistance = 100;\n",
       "    const fCharge = -1000;\n",
       "    const linkStrength = 0.7;\n",
       "    const collideStrength = 1;\n",
       "\n",
       "    // Simulation defined with variables\n",
       "    var simulation = d3.forceSimulation()\n",
       "        .force(\"link\", d3.forceLink()\n",
       "            .distance(linkDistance)\n",
       "            .strength(linkStrength)\n",
       "        )\n",
       "        .force(\"collide\", d3.forceCollide()\n",
       "            .radius(function (d) {\n",
       "                return d.r + 10\n",
       "            })\n",
       "            .strength(collideStrength)\n",
       "        )\n",
       "        .force(\"charge\", d3.forceManyBody()\n",
       "            .strength(fCharge)\n",
       "        )\n",
       "        .force(\"center\", d3.forceCenter(width / 2, height / 2))\n",
       "        .force(\"y\", d3.forceY(0))\n",
       "        .force(\"x\", d3.forceX(0));\n",
       "\n",
       "    // Pin down functionality\n",
       "    var node_drag = d3.drag()\n",
       "        .on(\"start\", dragstarted)\n",
       "        .on(\"drag\", dragged)\n",
       "        .on(\"end\", dragended);\n",
       "\n",
       "    function dragstarted(d) {\n",
       "        if (!d3.event.active) simulation.alphaTarget(0.3).restart();\n",
       "        d.fx = d.x;\n",
       "        d.fy = d.y;\n",
       "    }\n",
       "\n",
       "    function dragged(d) {\n",
       "        d.fx = d3.event.x;\n",
       "        d.fy = d3.event.y;\n",
       "    }\n",
       "\n",
       "    function dragended(d) {\n",
       "        if (!d3.event.active) simulation.alphaTarget(0);\n",
       "    }\n",
       "\n",
       "    function releasenode(d) {\n",
       "        d.fx = null;\n",
       "        d.fy = null;\n",
       "    }\n",
       "\n",
       "    //END Pin down functionality\n",
       "\n",
       "\n",
       "    /**\n",
       "     * Gets the best name for a node object\n",
       "     * @param {object} nodeData object\n",
       "     * @returns {str} canonical name of the node\n",
       "     */\n",
       "    function getCanonicalName(nodeData) {\n",
       "        if (nodeData.cname) {\n",
       "            return nodeData.cname;\n",
       "        } else if (nodeData.name) {\n",
       "            return nodeData.name\n",
       "        } else if (nodeData.bel) {\n",
       "            return nodeData.bel\n",
       "        } else {\n",
       "            console.log('Undefined node: ' + nodeData);\n",
       "            return 'UNDEFINED'\n",
       "        }\n",
       "    }\n",
       "\n",
       "    const color_circunferencia = \"black\";\n",
       "    const default_link_color = \"#AAAAAA\";\n",
       "    const nominal_base_node_size = 8;\n",
       "\n",
       "    // Normal and highlighted stroke of the links (double the width of the link when highlighted)\n",
       "    const nominal_stroke = 1.5;\n",
       "\n",
       "    // Zoom variables\n",
       "    const min_zoom = 0.1;\n",
       "    const max_zoom = 10;\n",
       "\n",
       "    var svg = d3.select(chart).append(\"svg\")\n",
       "        .attr(\"width\", width)\n",
       "        .attr(\"height\", height);\n",
       "\n",
       "    // // Create definition for arrowhead.\n",
       "    svg.append(\"defs\").append(\"marker\")\n",
       "        .attr(\"id\", \"arrowhead\")\n",
       "        .attr(\"viewBox\", \"0 -5 10 10\")\n",
       "        .attr(\"refX\", 20)\n",
       "        .attr(\"refY\", 0)\n",
       "        .attr(\"markerUnits\", \"strokeWidth\")\n",
       "        .attr(\"markerWidth\", 6)\n",
       "        .attr(\"markerHeight\", 6)\n",
       "        .attr(\"orient\", \"auto\")\n",
       "        .append(\"path\")\n",
       "        .attr(\"d\", \"M0,-5L10,0L0,5\");\n",
       "\n",
       "    // // Create definition for stub.\n",
       "    svg.append(\"defs\").append(\"marker\")\n",
       "        .attr(\"id\", \"stub\")\n",
       "        .attr(\"viewBox\", \"-1 -5 2 10\")\n",
       "        .attr(\"refX\", 15)\n",
       "        .attr(\"refY\", 0)\n",
       "        .attr(\"markerUnits\", \"strokeWidth\")\n",
       "        .attr(\"markerWidth\", 6)\n",
       "        .attr(\"markerHeight\", 6)\n",
       "        .attr(\"orient\", \"auto\")\n",
       "        .append(\"path\")\n",
       "        .attr(\"d\", \"M 0,0 m -1,-5 L 1,-5 L 1,5 L -1,5 Z\");\n",
       "\n",
       "    // Background\n",
       "    svg.append(\"rect\")\n",
       "        .attr(\"width\", \"100%\")\n",
       "        .attr(\"height\", \"100%\")\n",
       "        .attr(\"fill\", \"#ffffff\")\n",
       "        .style(\"pointer-events\", \"all\")\n",
       "        // Zoom + panning functionality\n",
       "        .call(d3.zoom()\n",
       "            .scaleExtent([min_zoom, max_zoom])\n",
       "            .on(\"zoom\", zoomed))\n",
       "        .on(\"dblclick.zoom\", null);\n",
       "\n",
       "\n",
       "    function zoomed() {\n",
       "        g.attr(\"transform\", d3.event.transform);\n",
       "    }\n",
       "\n",
       "    // g = svg object where the graph will be appended\n",
       "    var g = svg.append(\"g\");\n",
       "\n",
       "    var linkedByIndex = {};\n",
       "    graph.links.forEach(function (d) {\n",
       "        linkedByIndex[d.source + \",\" + d.target] = true;\n",
       "    });\n",
       "\n",
       "    function isConnected(a, b) {\n",
       "        return linkedByIndex[a.index + \",\" + b.index] || linkedByIndex[b.index + \",\" + a.index] || a.index == b.index;\n",
       "    }\n",
       "\n",
       "    function ticked() {\n",
       "        link.attr(\"x1\", function (d) {\n",
       "            return d.source.x;\n",
       "        })\n",
       "            .attr(\"y1\", function (d) {\n",
       "                return d.source.y;\n",
       "            })\n",
       "            .attr(\"x2\", function (d) {\n",
       "                return d.target.x;\n",
       "            })\n",
       "            .attr(\"y2\", function (d) {\n",
       "                return d.target.y;\n",
       "            });\n",
       "\n",
       "        node\n",
       "            .attr(\"transform\", function (d) {\n",
       "                return \"translate(\" + d.x + \", \" + d.y + \")\";\n",
       "            });\n",
       "    }\n",
       "\n",
       "\n",
       "    simulation\n",
       "        .nodes(graph.nodes)\n",
       "        .on(\"tick\", ticked);\n",
       "\n",
       "    simulation.force(\"link\")\n",
       "        .links(graph.links);\n",
       "\n",
       "    // Definition of links nodes text...\n",
       "\n",
       "    var link = g.selectAll(\".link\")\n",
       "        .data(graph.links)\n",
       "        .enter().append(\"line\")\n",
       "        .style(\"stroke-width\", nominal_stroke)\n",
       "        .style(\"stroke\", default_link_color)\n",
       "        .style(\"stroke-dasharray\", function (d) {\n",
       "            if (['decreases', 'directlyDecreases', 'increases', 'directlyIncreases', 'negativeCorrelation',\n",
       "                'positiveCorrelation'].indexOf(d.relation) >= 0) {\n",
       "                return \"none\"\n",
       "            } else {\n",
       "                return \"4, 4\"\n",
       "            }\n",
       "        })\n",
       "        .attr(\"marker-start\", function (d) {\n",
       "            if ('positiveCorrelation' == d.relation) {\n",
       "                return \"url(#arrowhead)\"\n",
       "            }\n",
       "            else if ('negativeCorrelation' == d.relation) {\n",
       "                return \"url(#stub)\"\n",
       "            }\n",
       "            else {\n",
       "                return \"\"\n",
       "            }\n",
       "        })\n",
       "        .attr(\"marker-end\", function (d) {\n",
       "            if (['increases', 'directlyIncreases', 'positiveCorrelation', 'isA', 'partOf'].indexOf(d.relation) >= 0) {\n",
       "                return \"url(#arrowhead)\"\n",
       "            } else if (['decreases', 'directlyDecreases', 'negativeCorrelation'].indexOf(d.relation) >= 0) {\n",
       "                return \"url(#stub)\"\n",
       "            } else {\n",
       "                return \"\"\n",
       "            }\n",
       "        });\n",
       "\n",
       "    var node = g.selectAll(\".nodes\")\n",
       "        .data(graph.nodes)\n",
       "        .enter().append(\"g\")\n",
       "        .attr(\"class\", \"node\")\n",
       "        // Next two lines -> Pin down functionality\n",
       "        .on('dblclick', releasenode)\n",
       "        .call(node_drag);\n",
       "\n",
       "    var circle = node.append(\"path\")\n",
       "        .attr(\"d\", d3.symbol()\n",
       "            .size(function (d) {\n",
       "                return Math.PI * Math.pow(size(d.size) || nominal_base_node_size, 2);\n",
       "            })\n",
       "        )\n",
       "        .attr(\"class\", function (d) {\n",
       "            return d.function\n",
       "        })\n",
       "        .style('fill', function (d) {\n",
       "            return function_colors[d.function]\n",
       "        })\n",
       "        .style(\"stroke-width\", nominal_stroke)\n",
       "        .style(\"stroke\", color_circunferencia);\n",
       "\n",
       "    var text = node.append(\"text\")\n",
       "        .attr(\"class\", \"node-name\")\n",
       "        // .attr(\"id\", nodehashes[d])\n",
       "        .attr(\"fill\", \"black\")\n",
       "        .attr(\"dx\", 12)\n",
       "        .attr(\"dy\", \".35em\")\n",
       "        .text(function (d) {\n",
       "            return getCanonicalName(d)\n",
       "        });\n",
       "\n",
       "    // Highlight on mouseenter and back to normal on mouseout\n",
       "    node.on(\"mouseenter\", function (d) {\n",
       "        set_highlight(d);\n",
       "    })\n",
       "        .on(\"mousedown\", function () {\n",
       "            d3.event.stopPropagation();\n",
       "        }).on(\"mouseout\", function () {\n",
       "        exit_highlight();\n",
       "    });\n",
       "\n",
       "    function exit_highlight() {\n",
       "        highlight_node = null;\n",
       "        if (focus_node === null) {\n",
       "            if (highlight_node_boundering != color_circunferencia) {\n",
       "                circle.style(\"stroke\", color_circunferencia);\n",
       "                text.style(\"fill\", \"black\");\n",
       "                link.style(\"stroke\", default_link_color);\n",
       "            }\n",
       "        }\n",
       "    }\n",
       "\n",
       "    function set_highlight(d) {\n",
       "        if (focus_node !== null) d = focus_node;\n",
       "        highlight_node = d;\n",
       "\n",
       "        if (highlight_node_boundering != color_circunferencia) {\n",
       "            circle.style(\"stroke\", function (o) {\n",
       "                return isConnected(d, o) ? highlight_node_boundering : color_circunferencia;\n",
       "            });\n",
       "            text.style(\"fill\", function (o) {\n",
       "                return isConnected(d, o) ? highlight_text : \"black\";\n",
       "            });\n",
       "            link.style(\"stroke\", function (o) {\n",
       "                return o.source.index == d.index || o.target.index == d.index ? highlighted_link_color : default_link_color;\n",
       "            });\n",
       "        }\n",
       "    }\n",
       "\n",
       "\n",
       "    // Freeze the graph when space is pressed\n",
       "    function freezeGraph() {\n",
       "        // Space button Triggers STOP\n",
       "        if (d3.event.keyCode == 32) {\n",
       "            simulation.stop();\n",
       "        }\n",
       "    }\n",
       "\n",
       "    // Call freezeGraph when a key is pressed, freezeGraph checks whether this key is \"Space\" that triggers the freeze\n",
       "    d3.select(window).on(\"keydown\", freezeGraph);\n",
       "}\n",
       "\n",
       "require.config({\n",
       "    paths: {\n",
       "        d3: '//cdnjs.cloudflare.com/ajax/libs/d3/4.5.0/d3.min'\n",
       "    }\n",
       "});\n",
       "\n",
       "var elementInnerHTML = \"<div id='oejdrchtvqxskazm'></div>\";\n",
       "\n",
       "element.append(elementInnerHTML);\n",
       "\n",
       "require(['d3'], function (d3) {\n",
       "    return init_d3_force(d3, {\"directed\": true, \"graph\": {\"annotation_list\": {\"BRCO\": [\"Cortical Neuron\", \"Spinal motor neuron\"], \"Condition\": [\"Normal Healthy State\"], \"PDStage\": [\"Early_onset_PD\", \"Familial_autosomal_dominant_PD\", \"Familial_autosomal_recessive_PD\", \"Idiopathic_PD\", \"Late_onset_PD\", \"Primary_PD\", \"Secondary_PD\", \"Sporadic_PD\"], \"Race\": [\"Caucasian\", \"Han Chinese in Singapore\", \"Han Chinese in Taiwan\", \"Italian\", \"Spanish\"], \"Transcriptionally_active_region\": [\"3'UTR\", \"exon 3\", \"exon 4\", \"exon 41\", \"exon 7\", \"intron 1\"], \"UserAnnotationCell\": [\"AIF-Deficient Cell\", \"osteosarcoma U2OS cells\"]}, \"annotation_pattern\": {}, \"annotation_url\": {\"Anatomy\": \"https://arty.scai.fraunhofer.de/artifactory/bel/annotation/anatomy/anatomy-20170511.belanno\", \"Cell\": \"https://arty.scai.fraunhofer.de/artifactory/bel/annotation/cell/cell-20170511.belanno\", \"CellLine\": \"https://arty.scai.fraunhofer.de/artifactory/bel/annotation/cell-line/cell-line-20170511.belanno\", \"Confidence\": \"https://arty.scai.fraunhofer.de/artifactory/bel/annotation/confidence/confidence-1.0.0.belanno\", \"Disease\": \"https://arty.scai.fraunhofer.de/artifactory/bel/annotation/disease/disease-20170511.belanno\", \"Gender\": \"https://arty.scai.fraunhofer.de/artifactory/bel/annotation/gender/gender-1.0.0.belanno\", \"MeSHAnatomy\": \"https://arty.scai.fraunhofer.de/artifactory/bel/annotation/mesh-anatomy/mesh-anatomy-20170511.belanno\", \"MeSHDisease\": \"https://arty.scai.fraunhofer.de/artifactory/bel/annotation/mesh-diseases/mesh-diseases-20170511.belanno\", \"Species\": \"https://arty.scai.fraunhofer.de/artifactory/bel/annotation/species-taxonomy-id/species-taxonomy-id-20170511.belanno\", \"Subgraph\": \"https://arty.scai.fraunhofer.de/artifactory/bel/annotation/neurommsig/neurommsig-1.0.1.belanno\"}, \"document_metadata\": {\"authors\": \"Alpha Tom Kodamullil, Reagon Karki, Nora Filep, Pranika Singh, Daniel Domingo-Fernandez, Amir Hossein Kayvanjoo, Akrishta Sahay, Arunabh Sharma, Lingling Xu\", \"contact\": \"alpha.tom.kodamullil@scai.fraunhofer.de,reagon.karki@scai-extern.fraunhofer.de\", \"copyright\": \"Copyright (c) 2012, rkarki. All Rights Reserved.\", \"description\": \"Parkinson's Disease Knowledge Assembly for AETIONOMY\", \"name\": \"Parkinson's Disease Knowledge Assembly\", \"version\": \"1.3.0\"}, \"namespace_pattern\": {\"HBP\": \".*\", \"dbSNP\": \"rs[0-9]+\"}, \"namespace_url\": {\"BRCO\": \"https://arty.scai.fraunhofer.de/artifactory/bel/namespace/brain-region-ontology/brain-region-ontology-1.0.0.belns\", \"CHEBI\": \"https://raw.githubusercontent.com/pharmacome/terminology/b46b65c3da259b6e86026514dfececab7c22a11b/external/chebi-names.belns\", \"DO\": \"https://arty.scai.fraunhofer.de/artifactory/bel/namespace/disease-ontology/disease-ontology-20170511.belns\", \"FlyBase\": \"https://arty.scai.fraunhofer.de/artifactory/bel/namespace/flybase/flybase-20170508.belns\", \"GOBP\": \"https://arty.scai.fraunhofer.de/artifactory/bel/namespace/go-biological-process/go-biological-process-20170511.belns\", \"GOBPID\": \"https://arty.scai.fraunhofer.de/artifactory/bel/namespace/go-biological-process-ids/go-biological-process-ids-20150601.belns\", \"GOCC\": \"https://arty.scai.fraunhofer.de/artifactory/bel/namespace/go-cellular-component/go-cellular-component-20170511.belns\", \"HGNC\": \"https://arty.scai.fraunhofer.de/artifactory/bel/namespace/hgnc-human-genes/hgnc-human-genes-20170511.belns\", \"MESH\": \"https://raw.githubusercontent.com/pharmacome/terminology/b46b65c3da259b6e86026514dfececab7c22a11b/external/mesh-names.belns\", \"MESHC\": \"https://arty.scai.fraunhofer.de/artifactory/bel/namespace/mesh-chemicals/mesh-chemicals-20170511.belns\", \"MESHCS\": \"https://arty.scai.fraunhofer.de/artifactory/bel/namespace/mesh-cell-structures/mesh-cell-structures-20170511.belns\", \"MESHD\": \"https://arty.scai.fraunhofer.de/artifactory/bel/namespace/mesh-diseases/mesh-diseases-20170511.belns\", \"MESHPP\": \"https://arty.scai.fraunhofer.de/artifactory/bel/namespace/mesh-processes/mesh-processes-20170511.belns\", \"MGI\": \"https://arty.scai.fraunhofer.de/artifactory/bel/namespace/mgi-mouse-genes/mgi-mouse-genes-20170511.belns\", \"NIFT\": \"https://arty.scai.fraunhofer.de/artifactory/bel/namespace/imaging-ontology/imaging-ontology-1.0.0.belns\", \"NTN\": \"https://arty.scai.fraunhofer.de/artifactory/bel/namespace/nutrition/nutrition-1.0.0.belns\", \"PDO\": \"https://arty.scai.fraunhofer.de/artifactory/bel/namespace/parkinson-disease-ontology/parkinson-disease-ontology-1.0.2.belns\", \"PTS\": \"https://arty.scai.fraunhofer.de/artifactory/bel/namespace/neurodegeneration-pathways/neurodegeneration-pathways-1.0.0.belns\", \"RGD\": \"https://arty.scai.fraunhofer.de/artifactory/bel/namespace/rgd-rat-genes/rgd-rat-genes-20170511.belns\", \"SCOMP\": \"https://arty.scai.fraunhofer.de/artifactory/bel/namespace/selventa-named-complexes/selventa-named-complexes-20170511.belns\", \"SDIS\": \"https://arty.scai.fraunhofer.de/artifactory/bel/namespace/selventa-legacy-diseases/selventa-legacy-diseases-20150601.belns\", \"SFAM\": \"https://arty.scai.fraunhofer.de/artifactory/bel/namespace/selventa-protein-families/selventa-protein-families-20170511.belns\"}, \"namespaces_uncached\": [], \"path\": \"/Users/danieldomingo/PycharmProjects/hbp/curation/parkinson/parkinsons.bel\", \"pybel_version\": \"0.13.1\"}, \"links\": [{\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"Subgraph\": {\"Caspase subgraph\": true}}, \"citation\": {\"name\": \"Dev Cell. 2011 Jul 19;21(1):92-101. doi: 10.1016/j.devcel.2011.06.017.\", \"reference\": \"21763611\", \"type\": \"PubMed\"}, \"evidence\": \"Mitochondria participate in apoptosis through a range of mechanisms that vary between vertebrates and invertebrates. In vertebrates, they release intermembrane space proteins, such as cytochrome c, to promote caspase activation in the cytosol. This process is the result of the loss of integrity of the outer mitochondrial membrane caused by proapoptotic members of the Bcl-2 family. This event is always accompanied by a fissioning of the organelle. Fission of mitochondria has also been reported to participate in apoptosis in Drosophila and Caenorhabditis elegans.\", \"key\": \"5c1d60a3ad747374f21c0dbe2dcda717785b7218678b275469bf993762306e3feb6a49bc986b7a95ba3c01de2775197ba667a8c99c406f267790925432573f22\", \"line\": 10467, \"relation\": \"association\", \"source\": 5, \"target\": 7}, {\"annotations\": {\"MeSHDisease\": {\"Parkinson Disease\": true}, \"Subgraph\": {\"Insulin signal transduction\": true}}, \"citation\": {\"name\": \"PLoS One. 2014 Sep 30;9(9):e108559. doi: 10.1371/journal.pone.0108559. eCollection 2014.\", \"reference\": \"25268761\", \"type\": \"PubMed\"}, \"evidence\": \"Several lines of evidence have demonstrated the important role of insulin-like growth factor-1 receptor (IGF1R) signaling pathway in the neuropathogenic process of cognitive impairment, while decreased Grb10-Interacting GYF Protein 2 (GIGYF2) expression can alter IGF1R trafficking and its downstream signaling pathways.\", \"key\": \"98e4de2e4a3150df2cc60be5e6270670ab01f5cd7b5dba69c4810b53b6c59ad7cfbd12854ac0bdfc1a3bfaf971d1159492443f9a2c3e11cf12eff8f5454accf7\", \"line\": 286, \"relation\": \"association\", \"source\": 6, \"target\": 10}, {\"annotations\": {\"MeSHDisease\": {\"Parkinson Disease\": true}, \"Subgraph\": {\"Insulin signal transduction\": true}}, \"citation\": {\"name\": \"PLoS One. 2014 Sep 30;9(9):e108559. doi: 10.1371/journal.pone.0108559. eCollection 2014.\", \"reference\": \"25268761\", \"type\": \"PubMed\"}, \"evidence\": \"Several lines of evidence have demonstrated the important role of insulin-like growth factor-1 receptor (IGF1R) signaling pathway in the neuropathogenic process of cognitive impairment, while decreased Grb10-Interacting GYF Protein 2 (GIGYF2) expression can alter IGF1R trafficking and its downstream signaling pathways.\", \"key\": \"01b853247f0b519fba70e6732b6c9493d74f9360a70b6c3cfc46523703d118e9ee9f74ea51d5aae92f9f8da985de65bcae48677ce5b01a70b68c2d7f208687d7\", \"line\": 287, \"relation\": \"association\", \"source\": 6, \"target\": 23}, {\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"Subgraph\": {\"Caspase subgraph\": true}}, \"citation\": {\"name\": \"Dev Cell. 2011 Jul 19;21(1):92-101. doi: 10.1016/j.devcel.2011.06.017.\", \"reference\": \"21763611\", \"type\": \"PubMed\"}, \"evidence\": \"Mitochondria participate in apoptosis through a range of mechanisms that vary between vertebrates and invertebrates. In vertebrates, they release intermembrane space proteins, such as cytochrome c, to promote caspase activation in the cytosol. This process is the result of the loss of integrity of the outer mitochondrial membrane caused by proapoptotic members of the Bcl-2 family. This event is always accompanied by a fissioning of the organelle. Fission of mitochondria has also been reported to participate in apoptosis in Drosophila and Caenorhabditis elegans.\", \"key\": \"b53950379e974e90d6a25e81c83afae60b40d4c17e87fcde4491b682a7643419ce81515bd7d6526a71aa9381b266a50275f4c8b1820544dd6f30c08c34a15a68\", \"line\": 10466, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"association\", \"source\": 7, \"target\": 15}, {\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"Subgraph\": {\"Caspase subgraph\": true}}, \"citation\": {\"name\": \"Dev Cell. 2011 Jul 19;21(1):92-101. doi: 10.1016/j.devcel.2011.06.017.\", \"reference\": \"21763611\", \"type\": \"PubMed\"}, \"evidence\": \"Mitochondria participate in apoptosis through a range of mechanisms that vary between vertebrates and invertebrates. In vertebrates, they release intermembrane space proteins, such as cytochrome c, to promote caspase activation in the cytosol. This process is the result of the loss of integrity of the outer mitochondrial membrane caused by proapoptotic members of the Bcl-2 family. This event is always accompanied by a fissioning of the organelle. Fission of mitochondria has also been reported to participate in apoptosis in Drosophila and Caenorhabditis elegans.\", \"key\": \"a73d3336032329542ec04c3e35ed0c87a681f613dc3c5f26e842f867099c4761671a4d6c65c7a0c11bb5a12b78c2de37a228d2cf8cd2911adf44728459daf615\", \"line\": 10467, \"relation\": \"association\", \"source\": 7, \"target\": 5}, {\"annotations\": {\"MeSHDisease\": {\"Fragile X Syndrome\": true}, \"Subgraph\": {\"FMR1 subgraph\": true}}, \"citation\": {\"name\": \" Neuron. 2008 Aug 28;59(4):634-47.\", \"reference\": \"18760699\", \"type\": \"PubMed\"}, \"evidence\": \"The fragile X mental retardation protein (FMRP) is an RNA-binding protein that controls translational efficiency and regulates synaptic plasticity. Here, we report that FMRP is involved in dopamine (DA) modulation of synaptic potentiation. AMPA glutamate receptor subtype 1 (GluR1) surface expression and phosphorylation in response to D1 receptor stimulation were reduced in cultured Fmr1(-/-) prefrontal cortex (PFC) neurons. Furthermore, D1 receptor signaling was impaired, accompanied by D1 receptor hyperphosphorylation at serine sites and subcellular redistribution of G protein-coupled receptor kinase 2 (GRK2) in both PFC and striatum of Fmr1(-/-) mice. FMRP interacted with GRK2, and pharmacological inhibition of GRK2 rescued D1 receptor signaling in Fmr1(-/-) neurons. Finally, D1 receptor agonist partially rescued hyperactivity and enhanced the motor function of Fmr1(-/-) mice. Our study has identified FMRP as a key messenger for DA modulation in the forebrain and may provide insights into the cellular and molecular mechanisms underlying fragile X syndrome.\", \"key\": \"70faa0a7378930641bc75fbf73cf10d7b4d459cbf0f7f19bf8054a1654b401b2f240c058a0ec4c2943be8f29d90e54d632f7d2841aa424337dce43167c97734c\", \"line\": 7809, \"relation\": \"association\", \"source\": 9, \"target\": 19}, {\"annotations\": {\"MeSHDisease\": {\"Parkinson Disease\": true}, \"Subgraph\": {\"Insulin signal transduction\": true}}, \"citation\": {\"name\": \"PLoS One. 2014 Sep 30;9(9):e108559. doi: 10.1371/journal.pone.0108559. eCollection 2014.\", \"reference\": \"25268761\", \"type\": \"PubMed\"}, \"evidence\": \"Several lines of evidence have demonstrated the important role of insulin-like growth factor-1 receptor (IGF1R) signaling pathway in the neuropathogenic process of cognitive impairment, while decreased Grb10-Interacting GYF Protein 2 (GIGYF2) expression can alter IGF1R trafficking and its downstream signaling pathways.\", \"key\": \"20621feed6170955115098580ebbb0203b67c3b3dd3768a69a70a40e3969fb13d1d186822edd1f239b44a864eb0395c8276ccc4d8da11d3e4e51124a60cdcfaf\", \"line\": 286, \"relation\": \"association\", \"source\": 10, \"target\": 6}, {\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"Subgraph\": {\"GRB10 subgraph\": true}}, \"citation\": {\"name\": \"PLoS One. 2014 Sep 30;9(9):e108559. doi: 10.1371/journal.pone.0108559. eCollection 2014.\", \"reference\": \"25268761\", \"type\": \"PubMed\"}, \"evidence\": \"Growth factor receptor-bound protein 10 (Grb10), a suppressor of IGF1R pathway, has been shown to play a critical role in regulating diabetes-associated cognitive impairment.\", \"key\": \"a3abb4db7727428e8730996aa42eb367904ba36e31f7ba380a3f8c1aabd52b20cc2db53c0e7c0753e47476cec808c8e465c73c84b50b92053ca4ef6e9a55a38d\", \"line\": 297, \"relation\": \"association\", \"source\": 10, \"target\": 20}, {\"annotations\": {\"Subgraph\": {\"Post-translational events\": true}}, \"citation\": {\"name\": \"Am J Med Genet A. 2012 Jul;158A(7):1654-61. doi: 10.1002/ajmg.a.35396. Epub 2012 Jun 7.\", \"reference\": \"22678932\", \"type\": \"PubMed\"}, \"evidence\": \"Rare copy number abnormalities of A2BP1 have been previously associated with cognitive impairment, attention deficit disorder and autism [Martin et al. (2007);\", \"key\": \"91b507332f0ef6fa6ebf1443680a1a67e98e9217aabb9f053f7361d9f35f52c6c9884475036769889aff63265a276a60bdbc74083598be22cfe51a26963af03b\", \"line\": 499, \"relation\": \"association\", \"source\": 10, \"target\": 33}, {\"key\": \"8369d40752f599ccb1db1379e08ab2bc8d1317ab26eefe5ff710990f185b140db93477741dbb4e4be321fde606fcd8900853152faed281a1f19bce66bc9661c8\", \"relation\": \"hasComponent\", \"source\": 11, \"target\": 22}, {\"key\": \"1efdc98698837934bc4b4d6587187b88893ae5edbdbc85f859f28898b06f06b5d4519f1124927e3f583b539ad0acbba5fb811571ca7a820b10a37d0cfadce57b\", \"relation\": \"hasComponent\", \"source\": 11, \"target\": 23}, {\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"Subgraph\": {\"Insulin signal transduction\": true}}, \"citation\": {\"name\": \"PLoS One. 2014 Sep 30;9(9):e108559. doi: 10.1371/journal.pone.0108559. eCollection 2014.\", \"reference\": \"25268761\", \"type\": \"PubMed\"}, \"evidence\": \"Accumulating evidence indicates that insulin-like growth factor-1 (IGF1) acts as a homeostatic modulator for normal brain functionality and synaptic plasticity, and the loss of IGF1 activity caused by diabetes may contribute to cognitive impairment [6]. Most biological functions of IGF1, including its anti-apoptotic function, are mediated by the insulin-like growth factor-1 receptor (IGF1R).\", \"key\": \"ec65014145702feae1520bc6c479b9826b9abbc6548564e732a25d74e3e2bcb7ddf3c1d48bc7a4a8febfec6d0f582e55e77bff715d68b0644ebbc892993f8e10\", \"line\": 323, \"relation\": \"decreases\", \"source\": 11, \"target\": 5}, {\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"Subgraph\": {\"Insulin signal transduction\": true}}, \"citation\": {\"name\": \"PLoS One. 2014 Sep 30;9(9):e108559. doi: 10.1371/journal.pone.0108559. eCollection 2014.\", \"reference\": \"25268761\", \"type\": \"PubMed\"}, \"evidence\": \"Accumulating evidence indicates that insulin-like growth factor-1 (IGF1) acts as a homeostatic modulator for normal brain functionality and synaptic plasticity, and the loss of IGF1 activity caused by diabetes may contribute to cognitive impairment [6]. Most biological functions of IGF1, including its anti-apoptotic function, are mediated by the insulin-like growth factor-1 receptor (IGF1R).\", \"key\": \"17f4019bd28bd34d7158ec50e689619d8458c3194b3625c3eb00b04e4fe9ff6bf66cd75236105c3407067a613b7f38a4bdf9f94249e343fa0672c857ffba44e4\", \"line\": 5705, \"relation\": \"increases\", \"source\": 22, \"target\": 9}, {\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"Subgraph\": {\"Insulin signal transduction\": true}}, \"citation\": {\"name\": \"PLoS One. 2014 Sep 30;9(9):e108559. doi: 10.1371/journal.pone.0108559. eCollection 2014.\", \"reference\": \"25268761\", \"type\": \"PubMed\"}, \"evidence\": \"Accumulating evidence indicates that insulin-like growth factor-1 (IGF1) acts as a homeostatic modulator for normal brain functionality and synaptic plasticity, and the loss of IGF1 activity caused by diabetes may contribute to cognitive impairment [6]. Most biological functions of IGF1, including its anti-apoptotic function, are mediated by the insulin-like growth factor-1 receptor (IGF1R).\", \"key\": \"2865d78e52bba7dffaddb9f554e87c947923ba1b081d10546ac2adc5959a7279cba7229c772ccc029814598a49fc9610c13c9f0ce90a64c43cb4cdebd62b258c\", \"line\": 5706, \"relation\": \"increases\", \"source\": 22, \"target\": 10}, {\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"Subgraph\": {\"Insulin signal transduction\": true}}, \"citation\": {\"name\": \"PLoS One. 2014 Sep 30;9(9):e108559. doi: 10.1371/journal.pone.0108559. eCollection 2014.\", \"reference\": \"25268761\", \"type\": \"PubMed\"}, \"evidence\": \"Accumulating evidence indicates that insulin-like growth factor-1 (IGF1) acts as a homeostatic modulator for normal brain functionality and synaptic plasticity, and the loss of IGF1 activity caused by diabetes may contribute to cognitive impairment [6]. Most biological functions of IGF1, including its anti-apoptotic function, are mediated by the insulin-like growth factor-1 receptor (IGF1R).\", \"key\": \"a1bb6bc8166f0f941778dce146acb501f22b3c456f945d58b3d124eb0df65824e319982c0f69dd5671b55093b3aa4f290cb3a0085e1ec2da0a367a99a95ec3a9\", \"line\": 5707, \"relation\": \"increases\", \"source\": 22, \"target\": 11}, {\"annotations\": {\"Confidence\": {\"Medium\": true}, \"Curator\": {\"NaN\": true}, \"INDRA_API\": {\"reach\": true}, \"INDRA_Belief\": {\"0.95\": true}, \"INDRA_UUID\": {\"220ab76b-9dd8-4895-8ded-072edad56fa1\": true}}, \"citation\": {\"reference\": \"11120746\", \"type\": \"PubMed\"}, \"evidence\": \"IGF-I elicited IGF-I receptor tyrosine phosphorylation, resulting in the specific activation of G (i2).\", \"key\": \"a72140c2b04b343917541ab5a915b9354cb3c8174a417d96bee42c715be574f36c78d1a7080ef59e6d7bb41160f3a1e8eaa7d3b56bfe3fc37da14f406ad38f7b\", \"line\": 26, \"relation\": \"increases\", \"source\": 22, \"target\": 26}, {\"key\": \"8010de486f899af17ecf2100464fb842a6ef0e7a89d8eaa4138bc404458c80d5e60342e779bd899a9be8a5b180bd9c9b23e61493194e2e8e25e2557b90906a7e\", \"relation\": \"hasVariant\", \"source\": 23, \"target\": 24}, {\"key\": \"00c3b3a9b0b20ab5973117dc576f4f277ee3fda6efc07a091e836df0678dbc1cd05c2dcb5ca4a961708c6c09ac6282793b45d5f05d48dc22408ef0aa68252be9\", \"relation\": \"hasVariant\", \"source\": 23, \"target\": 25}, {\"key\": \"7e21d5c90c0aa1d4bbc855518dbbb649d4cf91028bd512ec3d47d651765004813b1e977ad075280cf5b5910ac269b349de31826828c6a83305e20307df6701d8\", \"relation\": \"hasVariant\", \"source\": 23, \"target\": 26}, {\"key\": \"b877655bcf7657fa303e7e8f9b4471e132f177a011bc70aa0891caa4a6fc08de79f169389de16a94cea5c1cfea53447ff9f3b063b2d38e5ac95bc156d13081cc\", \"relation\": \"hasVariant\", \"source\": 23, \"target\": 27}, {\"annotations\": {\"MeSHDisease\": {\"Parkinson Disease\": true}, \"Subgraph\": {\"Insulin signal transduction\": true}}, \"citation\": {\"name\": \"PLoS One. 2014 Sep 30;9(9):e108559. doi: 10.1371/journal.pone.0108559. eCollection 2014.\", \"reference\": \"25268761\", \"type\": \"PubMed\"}, \"evidence\": \"Several lines of evidence have demonstrated the important role of insulin-like growth factor-1 receptor (IGF1R) signaling pathway in the neuropathogenic process of cognitive impairment, while decreased Grb10-Interacting GYF Protein 2 (GIGYF2) expression can alter IGF1R trafficking and its downstream signaling pathways.\", \"key\": \"8f666d7cc4396031633f8516c3bccc2cc3979fea52047df68253e8c8d5450cdc4e2fed8ea82f9fe63a4489bfde8904aba61a2cf874bd79d602e14a3a661893f3\", \"line\": 287, \"relation\": \"association\", \"source\": 23, \"target\": 6}, {\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"Subgraph\": {\"Insulin signal transduction\": true}}, \"citation\": {\"name\": \"J Biol Chem. 2010 Dec 31;285(53):42130-9. doi: 10.1074/jbc.M110.143412. Epub 2010 Oct 26.\", \"reference\": \"20980250\", \"type\": \"PubMed\"}, \"evidence\": \"In addition to inhibiting insulin receptor and IGF1R kinase activity by directly binding to the receptors, GRB10 can also negatively regulate insulin and IGF1 signaling by mediating insulin receptor and IGF1R degradation through ubiquitination.\", \"key\": \"b33294cfce3141f902aaece3c77da5ebd2f12a3b14e7618683a53a7317b098429818d32cfbc5369f09d22ed271d07a8448d29c748c52f35bc3e0501ded49b40c\", \"line\": 444, \"relation\": \"increases\", \"source\": 23, \"subject\": {\"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 6}, {\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"Subgraph\": {\"Insulin signal transduction\": true}}, \"citation\": {\"name\": \"PLoS One. 2014 Sep 30;9(9):e108559. doi: 10.1371/journal.pone.0108559. eCollection 2014.\", \"reference\": \"25268761\", \"type\": \"PubMed\"}, \"evidence\": \"Accumulating evidence indicates that insulin-like growth factor-1 (IGF1) acts as a homeostatic modulator for normal brain functionality and synaptic plasticity, and the loss of IGF1 activity caused by diabetes may contribute to cognitive impairment [6]. Most biological functions of IGF1, including its anti-apoptotic function, are mediated by the insulin-like growth factor-1 receptor (IGF1R).\", \"key\": \"a90fb62372b80c6ca8ac9ccc7103f84ebeeaac59fab00d12762487c3d7f1a8cf4a46be4941cde59fe0b0affcbdb84ee58cd59733649cba9e0e508287751d5d2f\", \"line\": 319, \"relation\": \"increases\", \"source\": 23, \"target\": 9}, {\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"Subgraph\": {\"Insulin signal transduction\": true}}, \"citation\": {\"name\": \"PLoS One. 2014 Sep 30;9(9):e108559. doi: 10.1371/journal.pone.0108559. eCollection 2014.\", \"reference\": \"25268761\", \"type\": \"PubMed\"}, \"evidence\": \"Accumulating evidence indicates that insulin-like growth factor-1 (IGF1) acts as a homeostatic modulator for normal brain functionality and synaptic plasticity, and the loss of IGF1 activity caused by diabetes may contribute to cognitive impairment [6]. Most biological functions of IGF1, including its anti-apoptotic function, are mediated by the insulin-like growth factor-1 receptor (IGF1R).\", \"key\": \"fd706b9876eccdc167c27f36b1d374f3dd8f1ecf16e039f26b9ef82d9dd3f4076ba5e6bb103b11911632e3d545720d5793c0d45cc7e4f9947a1073618b7daa02\", \"line\": 320, \"relation\": \"increases\", \"source\": 23, \"target\": 10}, {\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"Subgraph\": {\"Insulin signal transduction\": true}}, \"citation\": {\"name\": \"PLoS One. 2014 Sep 30;9(9):e108559. doi: 10.1371/journal.pone.0108559. eCollection 2014.\", \"reference\": \"25268761\", \"type\": \"PubMed\"}, \"evidence\": \"Accumulating evidence indicates that insulin-like growth factor-1 (IGF1) acts as a homeostatic modulator for normal brain functionality and synaptic plasticity, and the loss of IGF1 activity caused by diabetes may contribute to cognitive impairment [6]. Most biological functions of IGF1, including its anti-apoptotic function, are mediated by the insulin-like growth factor-1 receptor (IGF1R).\", \"key\": \"58a5eb371d1830da75ed1c989306ec7664c2739db0f3d0e0edf63d9019ab28d7560eb67a8228ab94c6b675524202570bd031395b76f6f6659c4dedb22746577d\", \"line\": 321, \"relation\": \"increases\", \"source\": 23, \"target\": 11}, {\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"Subgraph\": {\"Insulin signal transduction\": true}}, \"citation\": {\"name\": \"J Biol Chem. 2010 Dec 31;285(53):42130-9. doi: 10.1074/jbc.M110.143412. Epub 2010 Oct 26.\", \"reference\": \"20980250\", \"type\": \"PubMed\"}, \"evidence\": \"In addition to inhibiting insulin receptor and IGF1R kinase activity by directly binding to the receptors, GRB10 can also negatively regulate insulin and IGF1 signaling by mediating insulin receptor and IGF1R degradation through ubiquitination.\", \"key\": \"0499bc358b9063f57d1b2fe6b4cce5a63b138d8ee629914eeb82a9441af4c568b0ca11ad38431e45f83748e1759d346df855d5c5186fc3cd38792623b683654f\", \"line\": 442, \"object\": {\"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"relation\": \"decreases\", \"source\": 23, \"subject\": {\"modifier\": \"Degradation\"}, \"target\": 23}, {\"annotations\": {\"Confidence\": {\"Medium\": true}, \"Curator\": {\"Mohammed\": true}, \"INDRA_API\": {\"reach\": true}, \"INDRA_Belief\": {\"0.65\": true}, \"INDRA_UUID\": {\"5eaff6cc-ed44-4016-ba75-145ad0339f96\": true}}, \"citation\": {\"reference\": \"10397275\", \"type\": \"PubMed\"}, \"evidence\": \"IGF-1R may down-regulate the Fas expression rendering transformed ovarian mesothelial cells resistant to apoptosis.\", \"key\": \"b420242b1ad3df3fd397d1bc73f1781f9cc0a88575111e0f98ffffe332aa49fd457de1378f0bd4704c53d380049bed8b885cc52d05c8a484208ab54439aab84d\", \"line\": 3, \"relation\": \"decreases\", \"source\": 23, \"target\": 18}, {\"annotations\": {\"Confidence\": {\"Medium\": true}, \"Curator\": {\"Mohammed\": true}, \"INDRA_API\": {\"reach\": true}, \"INDRA_Belief\": {\"0.65\": true}, \"INDRA_UUID\": {\"dc18f789-8e40-4393-b71d-dff99f43a96a\": true}}, \"citation\": {\"reference\": \"10605001\", \"type\": \"PubMed\"}, \"evidence\": \"Heightened tyrosine phosphorylation of IRS-2 during granulocytic differentiation was not caused by an increase in expression of the tyrosine kinase IGF-I receptor, as measured by the amount of both the alpha- and beta-subunits.\", \"key\": \"b0e40b9c0aa2f733f7be0be4d79cca780647ab8ebde7e1dbff578794c8f434cb83cf9caa80bad57e691ee719644854556aa27dcc2e8e2ed15c4c11de246b3377\", \"line\": 13, \"relation\": \"increases\", \"source\": 23, \"target\": 29}, {\"annotations\": {\"Confidence\": {\"Medium\": true}, \"Curator\": {\"Mohammed\": true}, \"INDRA_API\": {\"reach\": true}, \"INDRA_Belief\": {\"0.86\": true}, \"INDRA_UUID\": {\"2ee72b17-b17d-465f-ac80-1a7b042fcaa5\": true}}, \"citation\": {\"reference\": \"10642432\", \"type\": \"PubMed\"}, \"evidence\": \"Taken together, these data illustrate that IGF-1R stimulates uPA production.\", \"key\": \"16e1176101a8b6d43761815014d07e7330412648d229c304591fb54732b39abd742be71146191755f1a509c2a06684e7a4af0a249b7fc3819334214fcb98c61f\", \"line\": 14, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 23, \"target\": 30}, {\"annotations\": {\"Confidence\": {\"Medium\": true}, \"Curator\": {\"Mohammed\": true}, \"INDRA_API\": {\"reach\": true}, \"INDRA_Belief\": {\"0.98\": true}, \"INDRA_UUID\": {\"d01a91e4-2b79-435f-985e-06dab75b288e\": true}}, \"citation\": {\"reference\": \"10747872\", \"type\": \"PubMed\"}, \"evidence\": \"We conclude that IGF-I and IGF-IR is able to mediate activation of STAT3 in vitro and in vivo and that JAKs are essential for the process of activation.\", \"key\": \"b5736e06d2087c4b6aa4b5fb89b579da3f1929e2d40798d526e4d5c7c0101083c553e2abc2a66b00ca3b45c1c222de572c428bb00fd4f6a037c7e1e284ce07eb\", \"line\": 15, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"regulates\", \"source\": 23, \"target\": 38}, {\"annotations\": {\"Confidence\": {\"Medium\": true}, \"Curator\": {\"Mohammed\": true}, \"INDRA_API\": {\"reach\": true}, \"INDRA_Belief\": {\"0.95\": true}, \"INDRA_UUID\": {\"5f739d7f-8617-420d-bea0-c83cea4a90e8\": true}}, \"citation\": {\"reference\": \"10786694\", \"type\": \"PubMed\"}, \"evidence\": \"At 39 degrees C, expression of the human IGF-IR in H19-7 cells induces an insulin like growth factor (IGF) I dependent differentiation.\", \"key\": \"2cfa042b2998daab4aefe01a4b827edfde0e52124da8afb84cba433a1187948c3a3feb192f709bf4158c47dcb71a53dadda129e4b169c91f73189ad5b249a904\", \"line\": 17, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 23, \"target\": 40}, {\"annotations\": {\"Confidence\": {\"Medium\": true}, \"Curator\": {\"Mohammed\": true}, \"INDRA_API\": {\"reach\": true}, \"INDRA_Belief\": {\"0.95\": true}, \"INDRA_UUID\": {\"ae9f8af6-7522-49e9-80bc-4b4849c01832\": true}}, \"citation\": {\"reference\": \"10851068\", \"type\": \"PubMed\"}, \"evidence\": \"However, IGF-IR signaling in the absence of IRS-1 can cause cell differentiation and growth arrest.\", \"key\": \"126735f12e8e1b87261dd05e184e6e77c5f734cfa97207136ed00d631f8f39c5ccf3cc51f4b60df76c92ee4e608aa6bb85e0a0bbe4090a55ad5df73b0a7b7fad\", \"line\": 18, \"relation\": \"increases\", \"source\": 23, \"target\": 3}, {\"annotations\": {\"Confidence\": {\"Medium\": true}, \"Curator\": {\"Mohammed\": true}, \"INDRA_API\": {\"reach\": true}, \"INDRA_Belief\": {\"0.86\": true}, \"INDRA_UUID\": {\"6cb4a2ce-4081-42f2-826e-61a8160bccd5\": true}}, \"citation\": {\"reference\": \"10889510\", \"type\": \"PubMed\"}, \"evidence\": \"Collectively, these results suggest that IGF-IR or Bcl-2 overexpression in neuroblastoma cells promotes cell survival by preventing mitochondrial membrane depolarization and caspase-3 activation, ultimately leading to increased tumor growth.\", \"key\": \"775fd93cb28af78464d36786704e4fd1c6595e1706e7e71e3976f4bd69e37006ad93665842ec67ea320a3faf0919f5ef125e1d3b468dbbb8883217158d7d90c8\", \"line\": 19, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"decreases\", \"source\": 23, \"target\": 15}, {\"annotations\": {\"Confidence\": {\"Medium\": true}, \"Curator\": {\"NaN\": true}, \"INDRA_API\": {\"sparser\": true}, \"INDRA_Belief\": {\"0.98\": true}, \"INDRA_UUID\": {\"72bd84ac-b8b1-4886-8f68-4f3f39321f73\": true}}, \"citation\": {\"reference\": \"10889510\", \"type\": \"PubMed\"}, \"evidence\": \"IGF-IR activation or overexpression of Bcl-2 in SHEP cells prevents mitochondrial membrane depolarization.\", \"key\": \"44d084b113af6d632ee6122035a7a57ffacb9b11d73ca0f9498bdc10bf263bc353f6e4a1c343bd1750f9399a617e8579435b92c9c2cf19c1cae7b77f12531db7\", \"line\": 20, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 23, \"target\": 14}, {\"annotations\": {\"Confidence\": {\"Medium\": true}, \"Curator\": {\"NaN\": true}, \"INDRA_API\": {\"reach\": true}, \"INDRA_Belief\": {\"0.95\": true}, \"INDRA_UUID\": {\"e336fa11-c031-40ea-8332-64a58a84b49e\": true}}, \"citation\": {\"reference\": \"11063741\", \"type\": \"PubMed\"}, \"evidence\": \"Using bicistronic constructs, we demonstrated here that the 5 ' untranslated region of the IGF-IR allows translation initiation by internal ribosome entry and therefore constitutes an internal ribosome entry site.\", \"key\": \"9afec775bf9206dfe70853a536838b95be581c2a990d73138262c4283d312147c7aa438256bef639a5212a7bbdc792bb0e29a55d4f210138f779b32f8a1efdf6\", \"line\": 23, \"relation\": \"increases\", \"source\": 23, \"target\": 1}, {\"annotations\": {\"Confidence\": {\"Medium\": true}, \"Curator\": {\"NaN\": true}, \"INDRA_API\": {\"sparser\": true}, \"INDRA_Belief\": {\"0.86\": true}, \"INDRA_UUID\": {\"7f789024-0a48-45af-89fb-b534c9457c49\": true}}, \"citation\": {\"reference\": \"11120746\", \"type\": \"PubMed\"}, \"evidence\": \"The present study determined whether in normal human intestinal smooth muscle cells the IGF-I receptor activates a heterotrimeric G protein and the role of G protein activation in mediating IGF-I-induced growth.\", \"key\": \"9db26e780e4e166abea459660eae2044cbaad816923b9a6c5a60f6778916a8d7a9765353eef049e4fa177efedf932ea57565bfe8be1dcb635842b06096c972ac\", \"line\": 27, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 23, \"target\": 12}, {\"annotations\": {\"Confidence\": {\"Medium\": true}, \"Curator\": {\"NaN\": true}, \"INDRA_API\": {\"reach\": true}, \"INDRA_Belief\": {\"0.86\": true}, \"INDRA_UUID\": {\"021f4a91-2dc8-4483-8385-cf44fc421f60\": true}}, \"citation\": {\"reference\": \"11250706\", \"type\": \"PubMed\"}, \"evidence\": \"Recently, IGF1R activation was shown to induce rapid and transient tyrosine dephosphorylation of FAK, p130 Crk associated substrate, and paxillin in MCF-7 breast cancer epithelial cells [XREF_BIBR].\", \"key\": \"9018b4bd1ae50102975995cbb1aacadc4091ea9016910ca6c5b583fee7b9751752e1fbbb610227ffbd5fcbbc8a9ea894aeef8a6c1156c584a5c57b4cc984288e\", \"line\": 31, \"relation\": \"decreases\", \"source\": 23, \"target\": 32}, {\"annotations\": {\"Confidence\": {\"Medium\": true}, \"Curator\": {\"NaN\": true}, \"INDRA_API\": {\"reach\": true}, \"INDRA_Belief\": {\"0.65\": true}, \"INDRA_UUID\": {\"94e9e515-0556-4dce-af3b-564c81b73bee\": true}}, \"citation\": {\"reference\": \"11278691\", \"type\": \"PubMed\"}, \"evidence\": \"The role of PI3K activity in the up-regulation of Id2 gene expression by the IGF-IR was confirmed by different methods and in different cell types.\", \"key\": \"57a0a470ac6a888bc18a8e3a9d6215bff5ba7264f44430b8977bc144ef63db845c2cb87f834b571aa754f1f86dfb0512a6cfd80a8e8f6ca09bbe1f57ba5e6742\", \"line\": 33, \"relation\": \"regulates\", \"source\": 23, \"target\": 21}, {\"annotations\": {\"Confidence\": {\"Medium\": true}, \"Curator\": {\"NaN\": true}, \"INDRA_API\": {\"reach\": true}, \"INDRA_Belief\": {\"0.94\": true}, \"INDRA_UUID\": {\"47da9fba-8c66-4435-974e-d026ce16a90c\": true}}, \"citation\": {\"reference\": \"11278917\", \"type\": \"PubMed\"}, \"evidence\": \"The ability of the IGF-IR to induce DNA repair was also demonstrated by infecting NWTb3 cells with UV irradiated adenovirus.\", \"key\": \"ff2170e72fcca2efcfd06337f028e51a4cf22971b0e238f546395bc0c4db1f7cba92564a611982ab8187ef3bb07968e59aa9ca8b22cecdb3eeb23fd66c607584\", \"line\": 34, \"relation\": \"increases\", \"source\": 23, \"target\": 0}, {\"annotations\": {\"Confidence\": {\"Medium\": true}, \"Curator\": {\"NaN\": true}, \"INDRA_API\": {\"reach\": true}, \"INDRA_Belief\": {\"0.95\": true}, \"INDRA_UUID\": {\"0de015a0-ba85-4f33-8881-c1f433879b7b\": true}}, \"citation\": {\"reference\": \"11461928\", \"type\": \"PubMed\"}, \"evidence\": \"Altogether these data indicate that scrapie infection affects the expression, binding affinity, and signal transduction mediated by the IGF-1R in neuroblastoma cells.\", \"key\": \"0e98caf4108e50893e808913d1c97e980396175d877c453e7f3ce3dcd332dd13c8df8eb94bd8f02daf9f0309b4feeb0e370cf60bbec31104cceb3c77db6a9daa\", \"line\": 40, \"relation\": \"increases\", \"source\": 23, \"target\": 2}, {\"annotations\": {\"Confidence\": {\"Medium\": true}, \"Curator\": {\"NaN\": true}, \"INDRA_API\": {\"reach\": true}, \"INDRA_Belief\": {\"0.92\": true}, \"INDRA_UUID\": {\"8572f013-d7f2-4497-a13a-77b272685734\": true}}, \"citation\": {\"reference\": \"11494131\", \"type\": \"PubMed\"}, \"evidence\": \"This suggests that IGF1R signalling can modulate the function of Atm, and supports the concept of targeted IGF1R downregulation as a potential treatment for malignant melanoma and other radioresistant tumours.\", \"key\": \"292cf5e9bd1333e7acccc122b1bfd93df0c45c20a58e1d9dc6483bd4053a25d28dee8e6689c19466ae5561383a1d2f71e61958ca8a948b8b13b4c775ad048eaf\", \"line\": 42, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 23, \"target\": 13}, {\"annotations\": {\"Confidence\": {\"Medium\": true}, \"Curator\": {\"NaN\": true}, \"INDRA_API\": {\"reach\": true}, \"INDRA_Belief\": {\"0.95\": true}, \"INDRA_UUID\": {\"cd506392-6a2e-4626-82d1-2a959a9a56d0\": true}}, \"citation\": {\"reference\": \"11559546\", \"type\": \"PubMed\"}, \"evidence\": \"Here we examined IGF-IR signaling and function in ER negative MDA-MB-231 breast cancer cells and their IGF-IR-overexpressing derivatives.\", \"key\": \"1e69bd8e56ac0575fd3345822f01d3e7a1382ca306995de23a389906ab53ec78df25cff9404e9167fdec40d633e273713246cadf9525e359d62fdc5913d010a9\", \"line\": 43, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 23, \"target\": 17}, {\"annotations\": {\"Confidence\": {\"Medium\": true}, \"Curator\": {\"NaN\": true}, \"INDRA_API\": {\"sparser\": true}, \"INDRA_Belief\": {\"0.98\": true}, \"INDRA_UUID\": {\"fb716a4b-e5f0-4a74-95a2-d1a0b805d1a0\": true}}, \"citation\": {\"reference\": \"11597519\", \"type\": \"PubMed\"}, \"evidence\": \"Therefore, it is interesting that GH can alter the phosphorylation status of ErbB-2, an EGF receptor-related protein that is overexpressed in about 25% of colorectal cancers, and that the IGF-IR can a[MISSING/INVALID API KEY: limited to 200 char for Elsev\", \"key\": \"13e98b44c37fbf9661e0a1cb9b64b7e9aad813870336fb0c925f600fdabc0291088f15e50e0766d53de94501b345b04b3b7cb6f0c3e47bb250fa6ef6a05ea439\", \"line\": 44, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 23, \"target\": 16}, {\"annotations\": {\"Confidence\": {\"Medium\": true}, \"Curator\": {\"NaN\": true}, \"INDRA_API\": {\"reach\": true}, \"INDRA_Belief\": {\"0.65\": true}, \"INDRA_UUID\": {\"79bdf7da-a9a1-41d4-a404-312d60c216c4\": true}}, \"citation\": {\"reference\": \"11693172\", \"type\": \"PubMed\"}, \"evidence\": \"These findings suggest that a reduction in the levels of IGF-1R induces a defective glandular homeostasis in the submandibular gland epithelial cells and triggers EAS.\", \"key\": \"ac31997c6b5b76a5b92efbab6983ddbb62a80ab375940e1f67b5f06d5758fdcebe3caeca0e5f8a5915c27250e2e3abaf9f6c050573e1b679727df646d142e988\", \"line\": 45, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"decreases\", \"source\": 23, \"target\": 41}, {\"annotations\": {\"Confidence\": {\"Medium\": true}, \"Curator\": {\"NaN\": true}, \"INDRA_API\": {\"reach\": true}, \"INDRA_Belief\": {\"0.65\": true}, \"INDRA_UUID\": {\"ee81b368-0051-4cd4-aec9-7fce7f3c2d8d\": true}}, \"citation\": {\"reference\": \"11693172\", \"type\": \"PubMed\"}, \"evidence\": \"These findings suggest that a reduction in the levels of IGF-1R induces a defective glandular homeostasis in the submandibular gland epithelial cells and triggers EAS.\", \"key\": \"ab82d183884ef6835f400c38aff8aa1fd8c6f946e953fd497e5b730dc9e823a15c547050fdea0d509bdf8b3e4473bd3fdb69f1bfba9d24ebfda0140b7f94826a\", \"line\": 46, \"relation\": \"decreases\", \"source\": 23, \"target\": 4}, {\"annotations\": {\"Confidence\": {\"Medium\": true}, \"Curator\": {\"NaN\": true}, \"INDRA_API\": {\"sparser\": true}, \"INDRA_Belief\": {\"0.94\": true}, \"INDRA_UUID\": {\"69430f2c-f026-4943-9969-d574252a6be0\": true}}, \"citation\": {\"reference\": \"11779860\", \"type\": \"PubMed\"}, \"evidence\": \"These studies indicate that IGF-IR stimulates phosphorylation of SHPS-1 which is critical for SHP-2 recruitment to the plasma membrane and for its recruitment to the IGF-IR.\", \"key\": \"7b758239ece31b561bbbdc10782731369d6e7dd9c13a19351b38f1d1af13c3257a0b0296acc51e6e3d0e554b7586ca88a9745e51bc1c51bd617619379c2b86b7\", \"line\": 47, \"relation\": \"increases\", \"source\": 23, \"target\": 36}, {\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"Subgraph\": {\"Bcl-2 subgraph\": true}}, \"citation\": {\"name\": \"Mol Cell. 2014 Aug 7;55(3):451-66. doi: 10.1016/j.molcel.2014.06.001. Epub 2014 Jul 3.\", \"reference\": \"24999239\", \"type\": \"PubMed\"}, \"evidence\": \"Bcl-2 family proteins regulate mitochondrial outer membrane permeabilization during apoptosis and can also influence macroautophagy via interactions with Beclin-1.\", \"key\": \"59465097f452b873fc788111b209239508f042210be82c95a95b4715e723f6c545a93c1a7062eba693fdbe0e1ba66f9ebaf31a6709cd0e359d97f70ce1540eab\", \"line\": 10403, \"relation\": \"increases\", \"source\": 14, \"target\": 8}, {\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"Subgraph\": {\"Caspase subgraph\": true}}, \"citation\": {\"name\": \"Dev Cell. 2011 Jul 19;21(1):92-101. doi: 10.1016/j.devcel.2011.06.017.\", \"reference\": \"21763611\", \"type\": \"PubMed\"}, \"evidence\": \"Mitochondria participate in apoptosis through a range of mechanisms that vary between vertebrates and invertebrates. In vertebrates, they release intermembrane space proteins, such as cytochrome c, to promote caspase activation in the cytosol. This process is the result of the loss of integrity of the outer mitochondrial membrane caused by proapoptotic members of the Bcl-2 family. This event is always accompanied by a fissioning of the organelle. Fission of mitochondria has also been reported to participate in apoptosis in Drosophila and Caenorhabditis elegans.\", \"key\": \"5ea23c4fa9788a473dbdffb6ca7e937a7f788b99f2631b54219c26f6440a3ba2b96e19549c9fd79f9272d516631c295a7dbf159caae93f0325f3c0ade26308a4\", \"line\": 10466, \"relation\": \"association\", \"source\": 15, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 7}, {\"annotations\": {\"Subgraph\": {\"Bcl-2 subgraph\": true, \"Caspase subgraph\": true, \"Mitochondrial subgraph\": true}}, \"citation\": {\"name\": \"FASEB J. 2002 Nov;16(13):1826-8. Epub 2002 Sep 5.\", \"reference\": \"12223445\", \"type\": \"PubMed\"}, \"evidence\": \"At nanomolar concentrations, a-SN protected neurons against serum deprivation, oxidative stress, and excitotoxicity through the PI3/Akt signaling pathway, and its protective effect was increased by Bcl-2 overexpression. Conversely, at both low micromolar and overexpressed levels in the cell, alpha-SN resulted in cytotoxicity. This might be related to decreased Bcl-xL expression and increased bax expression, which is subsequently followed by cytochrome c release and caspase activation and also by microglia-mediated inflammatory responses via the NFkappaB and mitogen-activated protein kinase pathways.\", \"key\": \"5351ffa966c3f49b860be675892fb036bbb5d8661488c0ca19bb17ca278f30c28ef0abd2f817cdd2eb5221d1be64e1f6781f1f1b32bd38e6277f6569f0e54ba1\", \"line\": 16945, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 15, \"target\": 37}, {\"annotations\": {\"MeSHDisease\": {\"Parkinson Disease\": true}, \"Subgraph\": {\"Calcium-dependent signal transduction\": true, \"Estrogen subgraph\": true}}, \"citation\": {\"name\": \"Endocrine. 2012 Aug;42(1):118-24. doi: 10.1007/s12020-012-9637-8. Epub 2012 Feb 28.\", \"reference\": \"22371119\", \"type\": \"PubMed\"}, \"evidence\": \"Estrogen receptors (ERÅ’Â± and ERÅ’â‰¤) mediate the neuroprotection of estrogens against MPTP-induced striatal dopamine (DA) depletion. Pain is an important and distressing symptom in Parkinson's disease (PD). Voltage-gated sodium channels in sensory neurons are involved in the development of neuropathic pain.The MPTP-induced up-regulation of Nav1.1 and Nav1.9, down-regulation of Nav1.6 in DRG neurons may be through ERÅ’â‰¤, up-regulation of Nav1.7 and down-regulation of Nav1.8 are dependent on both ERÅ’Â± and ERÅ’â‰¤.\", \"key\": \"cafa715313d8f308468085609f772f0b047e2faa67c97bed2f6e9e5c69d1d3de1aad9259baec85cb59a8a150b791a74cb2d3a07093318cc8b9717eb870a1cd58\", \"line\": 6663, \"relation\": \"decreases\", \"source\": 17, \"target\": 42}, {\"annotations\": {\"Confidence\": {\"Medium\": true}, \"Curator\": {\"NaN\": true}, \"INDRA_API\": {\"reach\": true}, \"INDRA_Belief\": {\"0.95\": true}, \"INDRA_UUID\": {\"661df270-4e79-4ac6-a8d0-6b35a68266db\": true}}, \"citation\": {\"reference\": \"10965916\", \"type\": \"PubMed\"}, \"evidence\": \"The dose of chemical necessary to activate IGF-I signaling varied, with the order of potency : E2 = diethylstilbestrol> LY353381> 4-hydroxytamoxifen> genistein> HPTE> bisphenol A. Administration of the chemicals to ERalpha knockout mice did not activate I\", \"key\": \"3084461a826aca6dafe93615c7a69eacc703db4a35cde0b3dacd0b165757a1d0ac6972583098ded3e1488423c2c0ff1b6125e306c408cb898484b7e812a1b9ec\", \"line\": 21, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 17, \"target\": 23}, {\"annotations\": {\"MeSHDisease\": {\"Fragile X Syndrome\": true}, \"Subgraph\": {\"FMR1 subgraph\": true}}, \"citation\": {\"name\": \" Neuron. 2008 Aug 28;59(4):634-47.\", \"reference\": \"18760699\", \"type\": \"PubMed\"}, \"evidence\": \"The fragile X mental retardation protein (FMRP) is an RNA-binding protein that controls translational efficiency and regulates synaptic plasticity. Here, we report that FMRP is involved in dopamine (DA) modulation of synaptic potentiation. AMPA glutamate receptor subtype 1 (GluR1) surface expression and phosphorylation in response to D1 receptor stimulation were reduced in cultured Fmr1(-/-) prefrontal cortex (PFC) neurons. Furthermore, D1 receptor signaling was impaired, accompanied by D1 receptor hyperphosphorylation at serine sites and subcellular redistribution of G protein-coupled receptor kinase 2 (GRK2) in both PFC and striatum of Fmr1(-/-) mice. FMRP interacted with GRK2, and pharmacological inhibition of GRK2 rescued D1 receptor signaling in Fmr1(-/-) neurons. Finally, D1 receptor agonist partially rescued hyperactivity and enhanced the motor function of Fmr1(-/-) mice. Our study has identified FMRP as a key messenger for DA modulation in the forebrain and may provide insights into the cellular and molecular mechanisms underlying fragile X syndrome.\", \"key\": \"2f110fed7ad94c357b755220e3934d4d423f0bed54694274bb84294265e4c5c4ca32be3c1651859ba1a10acef0f27a215481fe93f1604cb6bbf125a3f6c15838\", \"line\": 7809, \"relation\": \"association\", \"source\": 19, \"target\": 9}, {\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"Subgraph\": {\"GRB10 subgraph\": true, \"Insulin signal transduction\": true}}, \"citation\": {\"name\": \"PLoS One. 2014 Sep 30;9(9):e108559. doi: 10.1371/journal.pone.0108559. eCollection 2014.\", \"reference\": \"25268761\", \"type\": \"PubMed\"}, \"evidence\": \"Growth factor receptor-bound protein 10 (Grb10), a suppressor of IGF1R pathway, has been shown to play a critical role in regulating diabetes-associated cognitive impairment.\", \"key\": \"a4cead097e646b5a83ab2c1fb7de61ff9482348f1921a857f29d2ea0b3c5993bd6ebba5ed324d8113470bf6842bbd9572a3cb7cd053869149f5c53f526ad42fe\", \"line\": 294, \"relation\": \"decreases\", \"source\": 20, \"target\": 6}, {\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"Subgraph\": {\"GRB10 subgraph\": true}}, \"citation\": {\"name\": \"PLoS One. 2014 Sep 30;9(9):e108559. doi: 10.1371/journal.pone.0108559. eCollection 2014.\", \"reference\": \"25268761\", \"type\": \"PubMed\"}, \"evidence\": \"Growth factor receptor-bound protein 10 (Grb10), a suppressor of IGF1R pathway, has been shown to play a critical role in regulating diabetes-associated cognitive impairment.\", \"key\": \"b6b001d863af6235bc2a57947eef43a0d06b4934ae815af303e01a8edaf3df1b216b541c5178fcbc07df3199fcebe7b6b959e4e4585b9e99d6913ebc71003daa\", \"line\": 297, \"relation\": \"association\", \"source\": 20, \"target\": 10}, {\"annotations\": {\"MeSHDisease\": {\"Parkinson Disease\": true}, \"Subgraph\": {\"GRB10 subgraph\": true, \"Insulin signal transduction\": true}}, \"citation\": {\"name\": \"PLoS One. 2014 Sep 30;9(9):e108559. doi: 10.1371/journal.pone.0108559. eCollection 2014.\", \"reference\": \"25268761\", \"type\": \"PubMed\"}, \"evidence\": \"The interaction between BPS domain of Grb10 and IGF1R could directly inhibit IGF1R substrate phosphorylation [11], and decreased the levels of Grb10 expression below normal levels resulted in augmented IGF1 activation of Akt and ERK1/2 [9].\", \"key\": \"30548919a12954b59a5a63cb043d48b27c669c6bf49dfdf06c6eedcf824981c8697fa263ec093ac83b629df324daf87602e0cc5ccc69dc104f5e53b0a8f9cd77\", \"line\": 360, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"decreases\", \"source\": 20, \"target\": 22}, {\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"Subgraph\": {\"Insulin signal transduction\": true}}, \"citation\": {\"name\": \"PLoS One. 2014 Sep 30;9(9):e108559. doi: 10.1371/journal.pone.0108559. eCollection 2014.\", \"reference\": \"25268761\", \"type\": \"PubMed\"}, \"evidence\": \"Accumulating evidence indicates that insulin-like growth factor-1 (IGF1) acts as a homeostatic modulator for normal brain functionality and synaptic plasticity, and the loss of IGF1 activity caused by diabetes may contribute to cognitive impairment [6]. Most biological functions of IGF1, including its anti-apoptotic function, are mediated by the insulin-like growth factor-1 receptor (IGF1R).\", \"key\": \"a48a89093eb4fc1c0507e7e5861594ccdd24fcc0710b21ab9d2fe8cc86cd0a507b2f39236c5404bbcd8baacfc1798d2dcfadcc2fdb375ac89db85109c5cb4e38\", \"line\": 322, \"relation\": \"increases\", \"source\": 24, \"target\": 11}, {\"annotations\": {\"MeSHDisease\": {\"Parkinson Disease\": true}, \"Subgraph\": {\"Insulin signal transduction\": true}}, \"citation\": {\"name\": \"PLoS One. 2014 Sep 30;9(9):e108559. doi: 10.1371/journal.pone.0108559. eCollection 2014.\", \"reference\": \"25268761\", \"type\": \"PubMed\"}, \"evidence\": \"Studies have shown that growth factor receptor-bound protein 10 (Grb10) can interact with tyrosine-phosphorylated growth factor receptors such as IGF1R and insulin receptor (IR) and serve as an endogenous negative regulator of IGF1R signaling.\", \"key\": \"75275645c6af2847b53e75edbfb7f00225906e3d9748c7eb808aa3d784b20987815ff3716ab79a8810d369848cdd6006a8084bf712715dc0d1bc4214c04fe77a\", \"line\": 332, \"relation\": \"increases\", \"source\": 25, \"target\": 11}, {\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"Subgraph\": {\"Insulin signal transduction\": true}}, \"citation\": {\"name\": \"J Biol Chem. 2010 Dec 31;285(53):42130-9. doi: 10.1074/jbc.M110.143412. Epub 2010 Oct 26.\", \"reference\": \"20980250\", \"type\": \"PubMed\"}, \"evidence\": \"In addition to inhibiting insulin receptor and IGF1R kinase activity by directly binding to the receptors, GRB10 can also negatively regulate insulin and IGF1 signaling by mediating insulin receptor and IGF1R degradation through ubiquitination.\", \"key\": \"66cf5b4bfc6b5fbc3775704bea3caa29097b65837f170aacf0caf89c08a7be9719ca2e09cafbf5e0a6c1e86775713327cce5b55b64bc8a4d479334ae08259066\", \"line\": 440, \"object\": {\"modifier\": \"Degradation\"}, \"relation\": \"increases\", \"source\": 27, \"target\": 23}, {\"key\": \"621a2c826e8939d79b24e50a8b8f2eab3fa98eb7fe5ac738b91830c5c96b2719601059ee2311ef93030fbaaab731b76e434fd81f56669a58651ae5bb9dbd5450\", \"relation\": \"hasVariant\", \"source\": 28, \"target\": 29}, {\"key\": \"f92faac44f3e25e028d79ce0df8156a5b5d766b66b3ca14ac17aa2b220e747402e5828ebc3eb99ef774ac196e9ac049f2d845771783765b79e2607984eb1a810\", \"relation\": \"hasVariant\", \"source\": 31, \"target\": 32}, {\"annotations\": {\"Subgraph\": {\"Post-translational events\": true}}, \"citation\": {\"name\": \"Am J Med Genet A. 2012 Jul;158A(7):1654-61. doi: 10.1002/ajmg.a.35396. Epub 2012 Jun 7.\", \"reference\": \"22678932\", \"type\": \"PubMed\"}, \"evidence\": \"Rare copy number abnormalities of A2BP1 have been previously associated with cognitive impairment, attention deficit disorder and autism [Martin et al. (2007);\", \"key\": \"237a49b5cefd983b9074f5e78ed42cfc4f3eac2a87210a4cad29e44d23fd9f8fd5effdbcd61332e28b708fdf0fac00f36b7d296a400da10d42adaf673ac9d700\", \"line\": 499, \"relation\": \"association\", \"source\": 33, \"target\": 10}, {\"annotations\": {\"BRCO\": {\"Cortical Neuron\": true}, \"MeSHDisease\": {\"Parkinson Disease\": true}, \"Species\": {\"9606\": true}, \"Subgraph\": {\"Neurotrophic subgraph\": true, \"Transcription subgraph\": true}}, \"citation\": {\"name\": \"Biochim Biophys Acta. 2013 Dec;1833(12):2789-802. doi: 10.1016/j.bbamcr.2013.07.004. Epub 2013 Jul 18.\", \"reference\": \"23872421\", \"type\": \"PubMed\"}, \"evidence\": \"STAT3 was previously shown to be activated by oncogenic RET, independent of ligand and GFRâˆšÃ©Â¬Â±. In this study, we demonstrated that NRTN induced serine(727) but not tyrosine(705) phosphorylation of STAT3 in primary cortical neuron and neuronal cell lines. Remarkably, STAT3 phosphorylation was found to be mediated specifically by GFRâˆšÃ©Â¬Â±2c and RET9 isoforms.\", \"key\": \"c2c8e6de73488b8ad9a03f3958f0def29c94df930f8423e46c0a23bec98b0dcb52dadcccce04d7056d35f92d5b8b9edda2cb2978c974c5fbb89397f00d3cae69\", \"line\": 6285, \"relation\": \"increases\", \"source\": 34, \"target\": 39}, {\"key\": \"760baa87dd3a030ee0a87d297c24f00b145d5087f16f5cd93ab533638fa269011aa33a5f3ab1b7894adbd59f0f10c22d00132a1370f0ea8eb0d6e4551ea92694\", \"relation\": \"hasVariant\", \"source\": 35, \"target\": 36}, {\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"MeSHDisease\": {\"Parkinson Disease\": true}, \"Subgraph\": {\"Caspase subgraph\": true, \"Synuclein subgraph\": true}}, \"citation\": {\"name\": \"Brain Res. 2009 Feb 23;1256:19-33. doi: 10.1016/j.brainres.2008.12.020. Epub 2008 Dec 24.\", \"reference\": \"19135032\", \"type\": \"PubMed\"}, \"evidence\": \"In addition, Œ±-synuclein overexpression has been shown to trigger the reduction of caspase 3 activity, which, in turn, leads to lower levels of p53 immunoreactivity (Alves Da Costa et al., 2002).\", \"key\": \"71e12599771bbf7d2555c7b3328d4eea8ce32a9ac2270ce7669e15323664a37119b47bc9b6565cd8b8c439bd17e8f04b495436b68b150d18568ecc185285d78f\", \"line\": 4090, \"relation\": \"decreases\", \"source\": 37, \"target\": 15}, {\"key\": \"5ec7b24d1e725c66a499a42d496a85fed13bf7685a49d4265782725627375f8b82ef5093ace5f6f20b72875bc0ecb31cef0bc8cae354c6553725ed0aad22f162\", \"relation\": \"hasVariant\", \"source\": 38, \"target\": 39}, {\"annotations\": {\"BRCO\": {\"Cortical Neuron\": true}, \"MeSHDisease\": {\"Parkinson Disease\": true}, \"Species\": {\"9606\": true}, \"Subgraph\": {\"Neurotrophic subgraph\": true, \"Transcription subgraph\": true}}, \"citation\": {\"name\": \"Biochim Biophys Acta. 2013 Dec;1833(12):2789-802. doi: 10.1016/j.bbamcr.2013.07.004. Epub 2013 Jul 18.\", \"reference\": \"23872421\", \"type\": \"PubMed\"}, \"evidence\": \"STAT3 was previously shown to be activated by oncogenic RET, independent of ligand and GFRâˆšÃ©Â¬Â±. In this study, we demonstrated that NRTN induced serine(727) but not tyrosine(705) phosphorylation of STAT3 in primary cortical neuron and neuronal cell lines. Remarkably, STAT3 phosphorylation was found to be mediated specifically by GFRâˆšÃ©Â¬Â±2c and RET9 isoforms.\", \"key\": \"0a352e5186da1b4bcde47fdc395871ec7e8e76a418c1fdcd904a780434e4db70a1b04e74fcfba8b1475dee084ac2050bc8ce2419f7582657503878662391e0fa\", \"line\": 6279, \"relation\": \"increases\", \"source\": 38, \"target\": 34}], \"multigraph\": true, \"nodes\": [{\"function\": \"BiologicalProcess\", \"id\": \"76ec0f26f69b3ce3d3d60048e4a47523f0268619bc543fa2ee841f0d5a4b7af25a27c2420d89e30105fe25d57e0e97531be931da30872b9336672563325ef955\", \"name\": \"GO:0006281\", \"namespace\": \"GO\"}, {\"function\": \"BiologicalProcess\", \"id\": \"041efee11a78fb185c1c411429f51568dc60dc578ca60f489dc3c6d26994476f204d76e98ae42fde9480ed5e54841a130b837641b90ba282560ccccbadbab513\", \"name\": \"GO:0006412\", \"namespace\": \"GO\"}, {\"function\": \"BiologicalProcess\", \"id\": \"9255588be831bc1a4f6b68043fecb87817fe2c97a7f0b14b8a6ef0f7babbcdd8cfa536edb9a1c4ae3a8d8da08a451fca4844dd258d1421f41501afbb4b0216d2\", \"name\": \"GO:0007165\", \"namespace\": \"GO\"}, {\"function\": \"BiologicalProcess\", \"id\": \"906fa512d804f85a44a10461d7432790037a9945aa432a39242e6994872bddae4e8da5c401b52067fb8f04009cbeac7a4a44b66a124bac41c84161b6da179ba9\", \"name\": \"GO:0030154\", \"namespace\": \"GO\"}, {\"function\": \"BiologicalProcess\", \"id\": \"79432469c7fd77341530a073e74a17677076286b88839fa177e33b9aa2b8e38f966e4c1ef2065e69dff7e88137e26d23a74822d1af59cfd1d5ec91977403f34b\", \"name\": \"GO:0042592\", \"namespace\": \"GO\"}, {\"function\": \"BiologicalProcess\", \"id\": \"3b6274961b3ffdeb24d162fa8ee48994603d4263030d5015597037d4dcb0e80f2413b85d4648d2a8c56b68161162dcb95656e3637a245faa140d61a61232451d\", \"name\": \"apoptotic process\", \"namespace\": \"GOBP\"}, {\"function\": \"BiologicalProcess\", \"id\": \"2a030f16e6ae6a6e9faaf5c0ee94b2f6d1956840eab12808acc639cd5409bb85e691ff5a6553a692d7f32dd16604b86cbe427a54d99c5b833f62a0b62867a504\", \"name\": \"insulin-like growth factor receptor signaling pathway\", \"namespace\": \"GOBP\"}, {\"function\": \"BiologicalProcess\", \"id\": \"6cca08adf437e8429e20340e64906ca6e5bbd808e8a43d23e3b01a48b8fd3f9344bdc3859411334bb446f7c007ae5d085fabe75754f1c1cdba5dd3acf74527ff\", \"name\": \"mitochondrial fission\", \"namespace\": \"GOBP\"}, {\"function\": \"BiologicalProcess\", \"id\": \"81ad2aeafff93bf61ee701ff3e59af27dd394ba1a20ae855557198ede48930cc8bb15d7abe0826fabd6edb8d2369777b5408cd1fc145b2e3e5f180a56ce19b68\", \"name\": \"regulation of mitochondrial membrane permeability involved in apoptotic process\", \"namespace\": \"GOBP\"}, {\"function\": \"BiologicalProcess\", \"id\": \"20ae2e31a66e627912482dcd89c85cd4a7abdc174d25d479bf5935b69f385906a6e21e7ef6ca1503f584a3d31849b4ab2b00a5ac2ae696c877b6924acbdde86f\", \"name\": \"regulation of synaptic plasticity\", \"namespace\": \"GOBP\"}, {\"function\": \"BiologicalProcess\", \"id\": \"7085c04c66ff00d9ede4223662139766ae23124164f0ef453fdb23a5605c09df1809749b89f32471228956018a87d82e70ba03f41ce127eb1e49a4d7b1519884\", \"name\": \"cognition\", \"namespace\": \"GOBP\"}, {\"function\": \"Complex\", \"id\": \"ca623d45d64c3770161fc47036aa7dea295149f2f0e2878408f09c789de708bcf9beeb822891ebfe5bf4dc297ce640b895c9408719e2d1dbbfad0c236222c15f\", \"members\": [{\"function\": \"Protein\", \"id\": \"3369894fadb1dfa2a161a7804d9e5be75205b590f45176482fbb7013986d328fd7874e0e48874a6658bc4a2d90f06abaef588ce4071f2e3b42a29a7376a4e892\", \"name\": \"IGF1\", \"namespace\": \"HGNC\"}, {\"function\": \"Protein\", \"id\": \"b31afcae01c7f92d5d2820a46e90bac3095691e49f1760c5d8cd5c6f6cce073ab81487a24e56a4bd8fa79a232691c26d4f016410e2d8be0d58024eaf77c8a85b\", \"name\": \"IGF1R\", \"namespace\": \"HGNC\"}]}, {\"function\": \"Protein\", \"id\": \"a9e0ddc6aecbfb75f850b86f363f0e60dbc525c49092429fd0c85d60a92b31be4b3409fd70559a21b127cb65544060ebdd457d0e33d786e4fd446ee62416f90a\", \"name\": \"G_protein\", \"namespace\": \"FPLX\"}, {\"function\": \"Protein\", \"id\": \"0e39acaa7c405cfe28541caabd43e0606e8c56a2123ace7d2883b1d31679fb34878153d5a361982f13671f434b40af997574f6abd3e04a955f3f9d601fe36582\", \"name\": \"ATM\", \"namespace\": \"HGNC\"}, {\"function\": \"Protein\", \"id\": \"b25decf9a9958e757d163c2d365729a1f86914126328ab0aabc90b2c29eb3a4617cd266432c4459745e2a1cd10176488cd1ba102d74f3869d82ad68257204b06\", \"name\": \"BCL2\", \"namespace\": \"HGNC\"}, {\"function\": \"Protein\", \"id\": \"3fc355e7c574083fad31aa9edc5d5685615c49fd58885c2fbd439371619da84fbe274bd3325695ab21c912b5ea592d8031ffa114a9147908170dc187400ba74c\", \"name\": \"CASP3\", \"namespace\": \"HGNC\"}, {\"function\": \"Protein\", \"id\": \"dcd21f91b4b3e8279a275c45587fe249a2e070eb2feb480a84b7d4a8ac14dcdca1ec2010b4de784666ac69771aee6d5036fac0f6074d87c997c24159d1504ee1\", \"name\": \"ERBB2\", \"namespace\": \"HGNC\"}, {\"function\": \"Protein\", \"id\": \"e3bd31293c5403b997b5337008759608563f6ae428718ee90f4d614db4eb658a7c4766f27fc82bfa03095977835d87dd4f8fa70918f6f4ad7f866cc67c8ca2c1\", \"name\": \"ESR1\", \"namespace\": \"HGNC\"}, {\"function\": \"Protein\", \"id\": \"a99820bac2cb084a3455167eba808b4411dc54afe7426f105ee58e1db7432309bf7a55e7433e4505708cc35fee332d0586550068b1ce0b6e0cd690a8628b3b9b\", \"name\": \"FAS\", \"namespace\": \"HGNC\"}, {\"function\": \"Protein\", \"id\": \"92ef023ef9fdb59c25163b00250b2819f4be183985e878a6c834400c7b1580c82ab62208e6653d3a061fbcc27ac7324d8277121a5569b688c33400f274e94a96\", \"name\": \"FMR1\", \"namespace\": \"HGNC\"}, {\"function\": \"Protein\", \"id\": \"dcfbaf3fd764c9fc6208f15c68e4d9f64b5a3ed846aa675995e6b2f89d618dc1b97b4eae6b5dc5dff35d359c0bc80d5e5cb2a42117b4cedba73e581ad8d49c2c\", \"name\": \"GRB10\", \"namespace\": \"HGNC\"}, {\"function\": \"Protein\", \"id\": \"8d9bd69c3e9857a29ecc60c73c6b4c3285b5c890ed888945e2d6fbc0bbbe063b83b14e9bb162974b7ee4106c189d7a7bb78de897e94cd1bc749133a5ccf37792\", \"name\": \"ID2\", \"namespace\": \"HGNC\"}, {\"function\": \"Protein\", \"id\": \"3369894fadb1dfa2a161a7804d9e5be75205b590f45176482fbb7013986d328fd7874e0e48874a6658bc4a2d90f06abaef588ce4071f2e3b42a29a7376a4e892\", \"name\": \"IGF1\", \"namespace\": \"HGNC\"}, {\"function\": \"Protein\", \"id\": \"b31afcae01c7f92d5d2820a46e90bac3095691e49f1760c5d8cd5c6f6cce073ab81487a24e56a4bd8fa79a232691c26d4f016410e2d8be0d58024eaf77c8a85b\", \"name\": \"IGF1R\", \"namespace\": \"HGNC\"}, {\"function\": \"Protein\", \"id\": \"c3a024ff21654c09d60e4ae558a6f4391d7a48dec095ab8dc309f97dd2e4beec63a853aac995fd294182f044813e044a5e3e116cd8ffa06f489b48fa36f6ef9b\", \"name\": \"IGF1R\", \"namespace\": \"HGNC\", \"variants\": [{\"identifier\": {\"name\": \"Ph\", \"namespace\": \"bel\"}, \"kind\": \"pmod\"}]}, {\"function\": \"Protein\", \"id\": \"1450aceacd0bf559fa8b436859f1cc0b3f4f0085bddf04ddad71580d10826d01fedf5e1c4b5c17f3ee591b3130cbc1e4f200fbfdb56cd1bf54f23661758846a3\", \"name\": \"IGF1R\", \"namespace\": \"HGNC\", \"variants\": [{\"code\": \"Thr\", \"identifier\": {\"name\": \"Ph\", \"namespace\": \"bel\"}, \"kind\": \"pmod\"}]}, {\"function\": \"Protein\", \"id\": \"32bc6dea3d35b3cf795463ecbbfceebe8442469a0c5242abf40f86b4f4e7de4b9ba279e1afbf1b4a4d9ee0547fa3b5c360d17e44c5538b55696b4e4ae267aa8e\", \"name\": \"IGF1R\", \"namespace\": \"HGNC\", \"variants\": [{\"code\": \"Tyr\", \"identifier\": {\"name\": \"Ph\", \"namespace\": \"bel\"}, \"kind\": \"pmod\"}]}, {\"function\": \"Protein\", \"id\": \"25227224b86a1fc638c94850b6b70761aadbaad4b48784fb9b742d585c2459469ef8e8395f24f806a0101d9b23d929d804cf4985b2798e621d527a3af7ce7fa0\", \"name\": \"IGF1R\", \"namespace\": \"HGNC\", \"variants\": [{\"identifier\": {\"name\": \"Ub\", \"namespace\": \"bel\"}, \"kind\": \"pmod\"}]}, {\"function\": \"Protein\", \"id\": \"79d56bc18f00ae9dc237f4fe3b47c271414f61a63d62f77eab15110a056ec86cbd7588b031acdd1278b86a2d38e089e56eeb356820facb352267cd257148ef2c\", \"name\": \"IRS2\", \"namespace\": \"HGNC\"}, {\"function\": \"Protein\", \"id\": \"132e24bd943802a35b053e430270497b4febe1d3e5c0a3c751f314bc6b9ec3f39121ca6c6476e45359b2754f0e91a712bd7fcdca10a18136ac2003440802a221\", \"name\": \"IRS2\", \"namespace\": \"HGNC\", \"variants\": [{\"code\": \"Tyr\", \"identifier\": {\"name\": \"Ph\", \"namespace\": \"bel\"}, \"kind\": \"pmod\"}]}, {\"function\": \"Protein\", \"id\": \"36d48bf455194ef16908a590071a7588ed4c8d7d355fa42099116bc3eca66634b9ceac258351d43a49ea542f5dd71e247c74ff1e81a7594bf21ddce4603b87a0\", \"name\": \"PLAU\", \"namespace\": \"HGNC\"}, {\"function\": \"Protein\", \"id\": \"b98bfd5d7c7414fb09dae0a2bf3c2c982cdfa62ea37e5bd8807eb94aef08db3729e46eeea0ca21b9bebadc63508a9a09b2ae34e14292a5a4df1dd894a71752e8\", \"name\": \"PTK2\", \"namespace\": \"HGNC\"}, {\"function\": \"Protein\", \"id\": \"14db05c98c5bcd5280c11cca3de68274886e0c7082b7dfb8604d9c91323a95917352f9eb6fe90e55ee50d31b118e6a79e7148561b934e28ea90f5abef2303c78\", \"name\": \"PTK2\", \"namespace\": \"HGNC\", \"variants\": [{\"identifier\": {\"name\": \"Ph\", \"namespace\": \"bel\"}, \"kind\": \"pmod\"}]}, {\"function\": \"Protein\", \"id\": \"2092becfeb7569759e7c7ddbc4bb790fa9fca8666b76f8d07c03cedd820a5648b96d1bdc9a76898903fbdf08ed4644bbf0b0d62111db77eccb873e4d310f7b7d\", \"name\": \"RBFOX1\", \"namespace\": \"HGNC\"}, {\"function\": \"Protein\", \"id\": \"0f34fbc31c8c497a1049de45d0f5cf4adff73686f43d2e8bd3b88eeff2ddb45f0d5313d3bbb3d1a0188632a20f1929bd43199e91cdc5019e6723bfe5727204c3\", \"name\": \"RET\", \"namespace\": \"HGNC\"}, {\"function\": \"Protein\", \"id\": \"6a6625d08effdb3ec52dc8e11aea6b0e036c90ffafdbfd5e4589657b2a00cc25e7a379bb5f5e496d99b71703ca2af898e5a1a44a3c323ff8a8699ae80de01246\", \"name\": \"SIRPA\", \"namespace\": \"HGNC\"}, {\"function\": \"Protein\", \"id\": \"cb8aa04d6a876d1fc3408c077307cee7e21a0f250cba4e93ab8c105decbb316ec419b78521607c639acaf2a69a89f94bdd1da666a9c036c9594012967f40c4d4\", \"name\": \"SIRPA\", \"namespace\": \"HGNC\", \"variants\": [{\"identifier\": {\"name\": \"Ph\", \"namespace\": \"bel\"}, \"kind\": \"pmod\"}]}, {\"function\": \"Protein\", \"id\": \"4345441d1f420dc0d830377737dba9467f21b1637ed3472a697534ce698f2566b780dd52cda47c96b782242b1bcfa70e9a6147fb3bb251d1f3e1e2a0bb7038f3\", \"name\": \"SNCA\", \"namespace\": \"HGNC\"}, {\"function\": \"Protein\", \"id\": \"3127bcd18ece964c797c6ae9fcf716d99ad750c12e3f8121f817efbbedc60af9639294ab059c32ef6101cef3ccd9abc934d90dc7ccdbe6dc3604773782f7bf97\", \"name\": \"STAT3\", \"namespace\": \"HGNC\"}, {\"function\": \"Protein\", \"id\": \"6336bd86d2c26fb635abfd9fbeda038e76986ff17846f9e80a7db717839ff76d3336d898df4248f65254441e52618b4313b270c443bcc46f3c921f33eb11252f\", \"name\": \"STAT3\", \"namespace\": \"HGNC\", \"variants\": [{\"code\": \"Ser\", \"identifier\": {\"name\": \"Ph\", \"namespace\": \"bel\"}, \"kind\": \"pmod\", \"pos\": 727}]}, {\"function\": \"Protein\", \"id\": \"a2f00920af5c1320633bc9d90f112764d44d87b60b69d1c40c5db54539725c7648565bd2134d6d32196702d70598e748b68d66cc2cb3c0165ab0b7501cffdb20\", \"name\": \"IPR022350\", \"namespace\": \"IP\"}, {\"function\": \"Protein\", \"id\": \"c463a0623130b601d717fe3286cf4e847174e268da2e54ed7e9f1d977e79bf6f125b8ce2773eb2bc89e47b94017c8ab1f06b1fc012b2e562fb57884372112313\", \"name\": \"O65323\", \"namespace\": \"UP\"}, {\"function\": \"RNA\", \"id\": \"0d5af91aaf38499c18b2ee8d43cdf6a28f708b039690589fe29f203c16bbdc402668160a6db160ce4e5efbebe70373aa10f159104e29556b6593082a7ae2d2dc\", \"name\": \"SCN10A\", \"namespace\": \"HGNC\"}]}, \"#oejdrchtvqxskazm\", 1000, 650, {'Protein': '#1F77B4', 'Pathology': '#FF7F0E', 'BiologicalProcess': '#2CA02C', 'miRNA': '#D62728', 'Complex': '#98DF8A', 'Composite': '#9467BD', 'Reaction': '#000000', 'Gene': '#FFBB78', 'Abundance': '#AEC7E8', 'RNA': '#FF9896'});\n",
       "});"
      ],
      "text/plain": [
       "<IPython.core.display.Javascript object>"
      ]
     },
     "execution_count": 6,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "igf1r_graph = pybel.from_json_path(IGF1R, check_version=False)\n",
    "pybel_jupyter.to_jupyter(igf1r_graph)"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.6.8"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
